frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesTopology.cy.ts

```diff
@@ -85,22 +96,61 @@ const initIntercepts = () => {
     },
     mockK8sResourceList([mockProjectK8sResource({})]),
   );
+  cy.intercept(
+    {
+      method: 'POST',
+      pathname: '/api/proxy/apis/v1beta1/jobs/test-pipeline',
+    },
+    mockJobKF({}),
+  );
 };
 
-describe('Pipeline details', () => {
-  it('Test topology renders', () => {
-    initIntercepts();
+describe('Pipeline topology', () => {
+  describe('Pipeline details', () => {
+    it('Test pipeline topology renders', () => {
+      initIntercepts();
+
+      pipelineDetails.visit('test-project', 'test-pipeline');
+
+      pipelineDetails.findTaskNode('print-msg').click();
+      pipelineDetails
+        .findTaskDrawer()
+        .findByText('$(tasks.random-num.results.Output)')
+        .should('exist');
+      pipelineDetails.findCloseDrawerButton().click();
+
+      pipelineDetails.findTaskNode('flip-coin').click();
+      pipelineDetails.findTaskDrawer().findByText('/tmp/outputs/Output/data').should('exist');
+    });
+  });
 
-    pipelineDetails.visit('test-project', 'test-pipeline');
+  describe('Pipeline run details', () => {
+    it('Test pipeline run topology renders', () => {
```

Comment:
I'm not sure if this test fits in with the scope of a file named "Topology", but I don't consider this a blocker. Wondering if it makes more sense in a standalone test suite just for the run details page.

Reply:
Run details and pipeline details are very similar and both implant the same topology view but with different drawers. That's why I kept them in the same spot. Because they share a lot of the same setup intercepts

Reply:
Got it. So the drawer of details belongs to the topology view. That makes sense.